src/foam/dao/IsolatedDAO.js

```diff
@@ -0,0 +1,66 @@
+/**
+ * @license
+ * Copyright 2022 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
+  package: 'foam.dao',
+  name: 'IsolatedDAO',
+  extends: 'foam.dao.ProxyDAO',
+
+  documentation: `
+    Stores a DAO operation but does not call the delegate until
+    commit() is called.
+  `,
+
+  classes: [
+    {
+      name: 'DAOOperation',
```

Comment:
Please see DOP

Reply:
you are using DOP below, so why do you have methodName here?

Reply:
Oh interesting, I was using StoreAndForwardDAO as a reference and didn't catch that